frontend/ui/widget/keyvaluepage.lua

```diff
@@ -150,6 +153,15 @@ function KeyValueItem:init()
                 self.show_value = RenderText:truncateTextByWidth(self.value, self.cface, frame_internal_width - key_w_rendered, true)
                 self.show_key = self.key
             end
+            -- allow for displaying the non-truncated texts with Hold
+            if Device:isTouchDevice() then
+                self.ges_events.Hold = {
```

Comment:
Looks like none in frontent/ui/widget/ uses it, so I assume it's only things in frontend/apps/ that should.